src/table/components/TablePaginationActions.jsx

```diff
@@ -46,24 +75,55 @@ export default function TablePaginationActions(props) {
     }
   };
 
+  const handleSelectPage = (event) => {
+    onPageChange(event, parseInt(event.target.value));
+  };
+
   const onFirstPage = page === 0;
   const onLastPage = page >= Math.ceil(count / rowsPerPage) - 1;
 
+  const options = [];
+  for(let i = 0; i < Math.ceil(count / rowsPerPage); i++){
+    options.push({value: i, label: i+1})
+  }
+
+  const thinTable = tableWidth < 650;
```

Comment:
Hmm we should discuss the order that the different components are removed. To me it makes more sense to have dropdown + next/previous than to have all buttons

Reply:
Yeah, and we should probably decide on one way to remove the components as well - I think how Li removed first/last might be neater.

Reply:
agree